frontend_lib/src/helper.js

```diff
@@ -449,13 +449,15 @@ export const parserStringToList = (string, separatorList = [',', ';', '\n']) =>
   return parsedString.split(',').filter(notEmptyString => notEmptyString !== '')
 }
 
-// INFO - GB - 2019-07-31 - This function check if the email has three parts arranged like somethig@something.somethig
+// INFO - GB - 2021-09-15 - This function checks if the string looks like an email (username@domain)
+// with a non empty username and a non-empty domain and without <, >, comma or semicolon
 export const checkEmailValidity = email => {
-  const parts = email.split('@')
-  if (parts.length !== 2) return false
-
-  const domainParts = parts[1].split('.')
-  return domainParts.length === 2
+  const firstPart = email.substr(0, email.lastIndexOf('@'))
+  const secondPart = email.substr(email.lastIndexOf('@') + 1)
+  return firstPart !== '' && !firstPart.includes(',') && !firstPart.includes(';') &&
+    !firstPart.includes('>') && !firstPart.includes('<') &&
+    secondPart !== '' && !secondPart.includes(',') && !secondPart.includes(';') &&
+    !secondPart.includes('>') && !secondPart.includes('<')
```

Comment:
```suggestion
  return firstPart !== '' && secondPart !== '' && !/[,;<>]/.test(email)
```

Reply:
The backend also forbids new line characters, so if you settle on forbidding it:
```suggestion
  return firstPart !== '' && secondPart !== '' && !/[,;<>\n]/.test(email)
```

Reply:
Fixed